src/js/jsx/search/SearchBar.jsx

```diff
@@ -445,6 +463,12 @@ define(function (require, exports, module) {
                     this.props.executeOption(id);
                     headlights.logEvent("search", filtersString, _.kebabCase(category));
                 }
+            } else {
+                if (payload && payload.noOptionsDefault) {
+                    this.props.executeOption(searchTerm, payload.noOptionsDefault().noOptionsExecute);
+                } else {
+                    return { dontCloseDialog: true };
```

Comment:
I am assuming you mean a `@return {{dontCloseDialog: boolean}=}` at line 418 which has now been added but this comment won't be outdated because the change was not made to line 470.